tldw_Server_API/app/core/AuthNZ/repos/federated_managed_grant_repo.py

```
@@ -0,0 +1,233 @@
+            if not row:
+                raise RuntimeError(
+                    "SQLite federated_managed_grants table is missing. "
+                    "Run the AuthNZ migrations/bootstrap (see "
+                    "'python -m tldw_Server_API.app.core.AuthNZ.initialize')."
+                )
+        except Exception as exc:
+            logger.error(f"FederatedManagedGrantRepo.ensure_tables failed: {exc}")
+            raise
+
+    @staticmethod
+    def _row_to_dict(row: Any) -> dict[str, Any]:
```

Ruff (0.15.6)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `row`

(ANN401)